### PR TITLE
(Feature) Generate downloaded file name dynamically

### PR DESCRIPTION
**Pull request by fvictorio**
This solves the first part of #269. The name of the downloaded file is generated using the current network and the address of the token.

### CODE REVIEW

**Comment by fvictorio** [comment]
@vbaranov Done.

File: src/components/stepFour/utils.js [hunk 1]
```diff
@@ -392,6 +399,26 @@ export function scrollToBottom() {
...
 }
 
+export function getDownloadName (tokenAddress) {
+  return new Promise((resolve, reject) => {
+    getWeb3((web3) => {
+      const whenNetworkName = getNetworkVersion(web3)
+        .then((networkId) => {
+          let networkName = getNetWorkNameById(networkId);
+
+          if (networkName === 'Unknown') {
```
[the remark this answers is not in the document]
Comment (15chrjef) [reply src/components/stepFour/utils.js]: I don't fully understand the problem at hand here, but I am sure there is a reasonable solution. Can either @fvictorio  or @vbaranov explain to me.

I would also extract 'unknown' as a constant.

[the remark this answers is not in the document]
Comment (fvictorio) [reply src/components/stepFour/utils.js]: `getNetWorkNameById` takes a networkId number and returns a string with the name of the network. If it doesn't know that networkId, it returns `'Unknown'`. If you want to use the networkId when the name is not known, you have to compare the returned value with the literal `'Unknown'`.

I don't mind moving `'Unknown'` to a constant, but then the other strings (the network names) should be moved too.

[the remark this answers is not in the document]
Comment (vbaranov) [reply src/components/stepFour/utils.js]: I've added chains directory i constants and changed `getNetWorkNameById ` to return `null` if network is undefined.

**Pull request merged**